packages/ocaml-variants/ocaml-variants.4.12.0+trunk/opam

```diff
@@ -46,8 +46,10 @@ build: [
     "LIBS=-static" {ocaml-option-static:installed}
     "--disable-warn-error"
   ]
-  [make "-j%{jobs}%" {os != "cygwin"} "world"]
-  [make "-j%{jobs}%" {os != "cygwin"} "world.opt"] {!ocaml-option-bytecode-only:installed}
+  [make "-j%{jobs}%" {os != "cygwin"}
+    "world" {ocaml-option-bytecode-only:installed}
+    "world.opt" {!ocaml-option-bytecode-only:installed}
```

Comment:
```suggestion
  [make "-j%{jobs}%" {os != "cygwin"} "defaultentry"]
```
or just:
```suggestion
  [make "-j%{jobs}%" {os != "cygwin"}
```
will work too.

Reply:
ohh, thanks a lot, that's much better. Committed.